app/assets/stylesheets/middleware_topology.css

```diff
@@ -0,0 +1,9 @@
+.middleware .kube-topology g.MiddlewareDatasource text.glyph {
+    font-size: 20px;
+    fill: #81B0C8;
+}
+
+.middleware .kube-topology g text.attached-label {
```

Comment:
looking, it's done differently than in the `container_topology_controller.js`, perhaps the `show_hide_names` method can be factored out to some "common JS something". For now, I will do the same as controller for containers does -> adding/removing the visible class and use the common css instead of adding style `display: block/none`

If I do that, then yes, this style can go away, otherwise it's needed, because w/o it the font size is 18px